src/redis/driver_impl.go

```diff
@@ -52,15 +54,41 @@ func checkError(err error) {
 	}
 }
 
+func GetClientTLSConfig(redisTlsCACerts string) (*tls.Config, error) {
+	// Get the SystemCertPool, continue with an empty pool on error
+	rootCAs, _ := x509.SystemCertPool()
+	if rootCAs == nil {
+		rootCAs = x509.NewCertPool()
+	}
+
+	// Read in the cert file
+	certs, err := ioutil.ReadFile(redisTlsCACerts)
+	if err != nil {
+		return nil, err
+	}
+
+	// Append our cert to the system pool
+	if ok := rootCAs.AppendCertsFromPEM(certs); !ok {
+		logger.Warnf("No certs appended, using system certs only")
+	}
+
+	// Trust the augmented cert pool in our client
+	return &tls.Config{
+		RootCAs: rootCAs,
+	}, nil
+}
+
 func NewClientImpl(scope stats.Scope, useTls bool, auth, redisSocketType, redisType, url string, poolSize int,
-	pipelineWindow time.Duration, pipelineLimit int, tlsConfig *tls.Config) Client {
```

Comment:
is it possible to do something simple like create a new [CertPool](https://pkg.go.dev/crypto/x509#CertPool) from the System cert pool:
```go
tls.Config.RootCAs = x509.SystemCertPool()
tls.Config.RootCAs.AppendCertsFromPEM(<pems from secret>)
```